fml/platform/fuchsia/message_loop_fuchsia.cc

```diff
@@ -5,13 +5,15 @@
 #include "flutter/fml/platform/fuchsia/message_loop_fuchsia.h"
 
 #include <lib/async-loop/default.h>
-#include <lib/async/cpp/task.h>
 #include <lib/zx/time.h>
 
 namespace fml {
 
 MessageLoopFuchsia::MessageLoopFuchsia()
-    : loop_(&kAsyncLoopConfigAttachToCurrentThread) {}
+    : loop_(&kAsyncLoopConfigAttachToCurrentThread){
+      auto handler = [this](async_dispatcher_t* dispatcher, async::Task* task, zx_status_t status) { RunExpiredTasksNow(); };
```

Comment:
A suggestion to better surface idle wakes. We can make RunExpiredTasksNow return the number of expired tasks. Then, if the task count is zero, we can log or assert so we know we woke up for nothing.

Reply:
I think this is a good idea but as it would affect all platforms I think it's out of scope for this PR.